packages/nest/README.md

```diff
@@ -114,15 +129,19 @@ import { FeatureClient } from '@openfeature/nestjs-sdk';
 export class OpenFeatureTestService {
   constructor(
     @FeatureClient() private defaultClient: Client,
-    @FeatureClient({ name: 'differentServer' }) private namedClient: Client,
-  ) {
-  }
+    @FeatureClient({ name: 'differentProvider' }) private namedClient: Client,
```

Comment:
I think we should rename this to `OpenFeatureClient` before we remove the `experimental`. I think just `FeatureClient` is a bit disjointed, since we have tended to either add the entire "OpenFeature" prefix, or leave it off. I think `OpenFeatureClient` would be best, because the client inteface is called `Client` so it won't clash, and it's consistent with the equivalent in [react](https://github.com/open-feature/js-sdk/blob/main/packages/react/README.md#multiple-providers-and-scoping) (`</OpenFeatureProvider>`).

Reply:
It has been `OpenFeatureClient` before, but [`OpenFeatureClient` is used for the implementation](https://github.com/open-feature/js-sdk/blob/3ad61cc0553c8f92eb512004001b6843e9208691/packages/server/src/client/open-feature-client.ts) of the `Client` and it is also exported from the Server SDK.

We could remove the export of `OpenFeatureClient` from the Web SDK, as no one should use and import this directly anyways I would say. I have seen it several times that this type was used instead of `Client` in projects.
If we remove this I would be happy calling the interface `OpenFeatureClient`here.

Reply:
What do you think @toddbaert?

Reply:
Sorry for the late reply - if we can remove that export, I'm fine with it!

Reply:
Hey @toddbaert, removing this should be considered a breaking change I would say.
I am not sure how we want to continue on this then. What do you think?

Reply:
hmm, well, if I understand correctly, it would be technically breaking, but I think it could be argued it was a misuse. All our doc recommends using the interface.

If we want to be really strict, we could consider it a break and include it with some others I have in mind in a 2.0.0 (meaning to ask you guys about this soon... there's a small list of breaking improvements I think might be worth considering)...